test_iscsi_3par.py

```diff
@@ -46,39 +46,33 @@ def test_full_blank_comp():
 
 
 def test_dedup_absent_comp_new():

```

Comment:
Add comment above each test case as its easy to undertsand.

Reply:
In newest commit, added comment for test_no_cpg_sanity()